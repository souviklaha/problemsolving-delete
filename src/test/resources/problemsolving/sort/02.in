13
4 3 8 9 -2 6 10 13 4 5 -9 0 -2