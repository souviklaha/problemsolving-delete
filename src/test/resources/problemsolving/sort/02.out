-9 -2 -2 0 3 4 4 5 6 8 9 10 13